modules/mobile/spec/request/legacy_claims_decision_request_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     before do
       FactoryBot.create(:evss_claim, id: 1, evss_id: 600_117_255, user_uuid: user.uuid)
-      Flipper.disable(:mobile_lighthouse_claims)
```

Comment:
this doesn't work when I try to use the `disable_actor` method so im going to leave to the ticket that switches everything over to those methods to figure out.

Reply:
Sounds good, yeah I'm not 100% sure how disable is configured